config/locales/en/course/assessment/assessments.yml

```diff
@@ -75,7 +75,8 @@ en:
           failure: 'Could not delete assessment: %{error}'
         authenticate:
           header: 'Input Password'
-          message: 'The assessment is locked, please input the password.'
+          assessment_not_unlocked: 'The assessment will be unlocked on %{start_at}.'
```

Comment:
This would be better written as `message_not_started` since below also uses `message_state`

Reply:
Done!